mlir_graphblas/src/include/GraphBLAS/GraphBLASUtils.h

```diff
@@ -51,8 +51,8 @@ static const llvm::StringSet<> monoid2{"any", "land", "lor",  "max",
 
 // TODO: create allUnary, allBinary, allMonoid
 
-static const llvm::StringSet<> supportedForUnion{"first", "max",    "min",
-                                                 "plus",  "second", "times"};
+static const llvm::StringSet<> supportedForUnion{
+    "first", "max", "min", "plus", "minus", "second", "times"};
```

Comment:
I would like to avoid including this. "minus" with a `union` is really surprising. I think the better approach is in two steps:
1. `apply` with "ainv"
2. `union` with "plus"

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/ce26f591efd6031e53380d12bb5641ce99a4d9bd implements this suggestion.